### DISCUSSION TITLE
Circle template of this skin

### DISCUSSION THREAD

**Discussion by NukeThief** [Saltssaumure/fnf-osu-mania-skin#17] [Q&A] [answered]
Basically i really want a template of this skin that has circles instead of arrows beacuse every other mania skin i download has a different way of coloring arrows, and my colors are different each that's why i need a template of this skin that has circles instead of arrows

**Comment by Saltssaumure** [comment, marked as answer]
In case you haven't already downloaded the latest version, a circle style is included.

**Reply by NukeThief** [reply]
Oh thank you
